database/guilds.go

```diff
@@ -33,186 +39,128 @@ func SelectGuild(f map[string]string) (models.Guild, error) {
 	return guild, nil
 }
 
-func InsertGuild(g string) error {
-	tx, err := db.Begin(context.Background())
+func InsertGuild(ctx context.Context, g string) error {
+	conn, err := pool.Acquire(ctx)
 	if err != nil {
 		return err
 	}
-	defer tx.Rollback(context.Background()) // Rollback the transaction if it hasn't been committed
+	defer conn.Release()
 
 	query := psql.Insert("guilds").Columns("guild_id").Values(g)
 	sql, args, err := query.ToSql()
 	if err != nil {
 		return err
 	}
 
-	commandTag, err := db.Exec(context.Background(), sql, args...)
+	tx, err := conn.Begin(ctx)
 	if err != nil {
 		return err
 	}
+	defer tx.Rollback(ctx)
 
-	if commandTag.RowsAffected() != 1 {
-		return fmt.Errorf("expected 1 row to be affected, got %d", commandTag.RowsAffected())
-	}
-
-	err = InitializeGuildCategories(g)
+	commandTagGuild, err := tx.Exec(ctx, sql, args...)
 	if err != nil {
 		return err
 	}
 
-	err = InitializeGuildBosses(g)
+	query = psql.Insert("guild_categories").
+		Columns("guild_id", "category", "message_id").
+		Suffix("SELECT $1, name, '' FROM categories", g)
+
+	sql, args, err = query.ToSql()
 	if err != nil {
 		return err
 	}
 
-	err = tx.Commit(context.Background())
+	commandTagCategories, err := tx.Exec(ctx, sql, args...)
 	if err != nil {
 		return err
 	}
 
-	return nil
-}
+	query = psql.Insert("guild_bosses").
```

Comment:
This query makes the /api/v1/guild {POST} route fail with a `409 Conflict`. With the error `Error creating guild: insert statements must have at least one set of values or select clause`

This fails when calling `.ToSql()` on it.